### ISSUE TITLE
Sails try to subscribe to the model when autosubsribe is false

### ISSUE THREAD

**Issue by heavyskyl** [balderdashy/sails#2455]
So if you have a model with autosubscribe: false, sails produce the warning in console:

"warn: `subscribe` called without context on a model with autosubscribe:false.  No action will be taken.".

I found this in the code:
sails/lib/hooks/blueprints/actionUtil.js

subscribeDeep function

instead of 'AssociatedModel.subscribe' I suppose to use
'if (AssociatedModel.autosubscribe) AssociatedModel.subscribe('

**Comment by CWyrtzen** [comment]
Is this still an issue with the latest release? Do we still need a failing test?

**Comment by sailsbot** [comment]
Thanks for posting, @heavyskyl.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
